contracts/validator-manager/ValidatorManager.sol

```diff
@@ -236,7 +233,7 @@ abstract contract ValidatorManager is Initializable, ContextUpgradeable, IValida
         if (input.blsPublicKey.length != BLS_PUBLIC_KEY_LENGTH) {
             revert InvalidBLSKeyLength(input.blsPublicKey.length);
         }
-        if (input.nodeID == bytes32(0)) {
+        if (input.nodeID.length == 0) {
```

Comment:
As of Etna, the P-Chain will only support 20 byte node IDs, but it's planned to add support for other node IDs in the future (particularly 32-byte Ed25519 node IDs).

I've gone back and forth on if we should require 20 byte node IDs here and track them as `bytes20`, or if we should leave it more open to support those future upgrades by default.